module-2/lab-df-calculation-and-transformation/your-code/challenge-3.ipynb

```diff
@@ -26,7 +26,7 @@
   },
```

Comment:
- Sub Probme 1 - Collecting Information: Your output is incorrect. Is it your code complete?